src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
```

Comment:
just add one default class `stars` in this case

Reply:
I tryed to do it, but result was not good